### DISCUSSION TITLE
Huge shoutout for this integration

### DISCUSSION THREAD

**Discussion by mjakeseaton** [dahlb/ha_hatch#13] [Show and tell]
Excellent work on this latest release. 

I am able to use node-red to automate my daughter's Hatch+ to turn on a specific rgb_color and brightness, and start the media player in our preferred sound_mode and volume_level. I can also turn off the entire device based on specific states. 

Thank you!

**Comment by michael-gaudet** [comment]
I'm curious, why is this not an official integration in HA? I did see some discussion of a pull request for Rest that went stale due to a lack of test coverage.

This has been a requested integration on HA for at least four years now. I might be willing to provide some financial motivation to get this to the goal line. 

For what it's worth, Ive tried contacting Hatch directly. While they've added Alexa integration, they have no intention of exposing any kind of API.

**Reply by dahlb** [reply]
reference the pr into core you were talking about seems to be https://github.com/home-assistant/core/pull/77381 using https://github.com/kjoconnor/pyhatchbabyrest/issues/7 library

that whole pr for rest is just thee BLE version of interacting with a rest, very different from what this integration does.

I'm not opposed to adding this integration to core but from what I've seen it will be a slow and long process that I'm not sure I have time for as I'm more concerned with it working then being in core vs hacs

**Comment by tmeekes** [comment]
Honestly, this is just so convenient to use. Having my Google Home system connected to HA, then HA connected to the Hatch has made it so easy to make voice commands AND build out automations for the kids sleep routines.

Thank you for the work you do on maintaining this.
